src/layout/IHyprLayout.cpp

```diff
@@ -552,10 +552,10 @@ void IHyprLayout::changeWindowFloatingMode(PHLWINDOW pWindow) {
 
         pWindow->m_vLastFloatingSize = PSAVEDSIZE;
```

Comment:
I thought about moving this line below `onWindowCreated(pWindow)` because it applies `pWindow->m_vLastFloatingSize` again, but it does not seem to matter, it is remembered anyway.

Reply:
ye